static/docs/get-started/import-data.md

```diff
@@ -0,0 +1,70 @@
+# Import Data
+
+We've seen how to [push](/doc/get-started/store-data) and
+[pull](/doc/get-started/retrieve-data) data a <abbr>DVC project</abbr>'s
+[remote](/doc/commands-reference/remote). But what if we wanted to integrate a
+dataset or ML model produced in one project into another project? (As an
+[external dependency](/doc/user-guide/external-dependencies))
+
+One way is to download the data (with `wget` or `dvc get`, for example) and use
+`dvc add` to track it locally, but the connection between projects would be
+lost: we couldn't tell where the data came from or whether there are new
+versions available. We offer the `dvc import` command instead (for projects
```

Comment:
`We offer` something less salesy? :)

Reply:
True. Rephrasing!

Reply:
p.s. Note that due to semantics I'm splitting this paragraph into 2 smaller ones. I know you only wanted 1 or 2 paragraphs but it just wouldn't be good writing. But it's the same number of words after all (:

Reply:
p.s. Actually I figured out a way to keep all this within 3 paragraphs:  by moving the last paragraph after the example up to the last one introducing `dvc import` (split from the one above in this review comment).